tasks/setup/vss-extension.json

```
@@ -0,0 +1,41 @@
+{
+  "manifestVersion": 1,
+  "id": "setup-infracost-task",
+  "name": "Setup Infracost",
+  "version": "0.0.15",
+  "publisher": "infracost",
```

Not sure if it matters, but our publisher has ID with capital "I" (and it can't be changed :D)
```suggestion
  "publisher": "Infracost",
```

---

done in https://github.com/infracost/infracost-azure-devops/pull/9/commits/95030bf9394c362cb56a6bdec90109b014fcd819